en/starter/generator/project_layout_reference.md

```diff
@@ -177,12 +177,12 @@ files will be included in the scaffolded project:
 │       └── Services/
 │            └── ServiceMongodb.swift - Swift connector to an IBM Cloud Service
 ├── manifest.yml                      - CloudFoundry deployment manifest file
-├── cli-config.yml                    - Bluemix CLI Dev plugin configuration file
+├── cli-config.yml                    - IBM Cloud CLI Dev plugin configuration file
```

Comment:
'IBM Cloud Dev Tools CLI configuration file' would better match the new naming convention.

Reply:
Noted, I'll open a new PR with the changes.